### PR TITLE
Updated install and admin guides to ovirt 4.2

### PR DESCRIPTION
**Pull request by bproffitt**
Changes proposed in this pull request:

- Updated Installation Guide to oVirt 4.2

- Updated Admin Guide to oVirt 4.2

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @bproffitt

This pull request needs review by: @gregsheremeta @sandrobonazzola

### CODE REVIEW

File: source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md [hunk 1]
```diff
@@ -8,24 +8,35 @@ title: Enterprise Linux Hosts
 
 An Enterprise Linux host (such as CentOS or RHEL), also known as an EL-based hypervisor is based on a standard basic installation of an Enterprise Linux operating system on a physical server.
 
-Ensure that you have required repositories for your distribution.
-On CentOS the Base, Optional and Extras repositories are already enabled by default and must be enabled.
-On Red Hat Enterprise Linux you'll need a valid subscription and the following repositories enabled:
+The host must meet the minimum Host requirements out lined in [Chapter 2: System Requirements](../chap-System_Requirements).
```
Comment (sandrobonazzola) [inline source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md]: I think that the section about needed repositories and yum install of ovirt-release42 rpm is still needed here

File: source/documentation/install-guide/chap-Introduction_to_Hosts.html.md [hunk 2]
```diff
@@ -1,38 +1,25 @@
...
-|-
-| 7.0 | ✓ |   |   |   |
-| 7.1 | ✓ |   |   |   |
-| 7.2 | ✓ | ✓ |   |   |
-| 7.3 |   | ✓ | ✓ |   |
-| 7.4 |   |   | ✓ | ✓ |
-| 7.5 |   |   |   | ✓ |
-
-When you create a new data center, you can set the compatibility version. Select the compatibility version that suits all the hosts in the data center. Once set, version regression is not allowed. For a fresh oVirt installation, the latest compatibility version is set in the Default Data Center and Default Cluster; to use an older compatibility version, you must create additional data centers and clusters.
+When you create a new data center, you can set the compatibility version. Select the compatibility version that suits all the hosts in the data center. Once set, version regression is not allowed. For a fresh oVirt installation, the latest compatibility version is set in the default data center and default cluster; to use an earlier compatibility version, you must create additional data centers and clusters.
```
Comment (sandrobonazzola) [inline source/documentation/install-guide/chap-Introduction_to_Hosts.html.md]: Is compatibility matrix not relevant anymore?

Comment (bproffitt) [reply source/documentation/install-guide/chap-Introduction_to_Hosts.html.md]: It was removed from the downstream documentation completely, so I'm going to go with no? I thought it drastic, too, but without solid change logs between RHV 4.0 and 4.2 documentation, I don't know when or why it was removed.

File: source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md (hunk 1 above)
Comment (bproffitt) [reply source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md]: Can upstream oVirt users get to them? Can you help with the appropriate text?

Comment (bproffitt) [reply source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md]: @sandrobonazzola Ping.

File: source/documentation/install-guide/chap-Introduction_to_Hosts.html.md (hunk 2 above)
Comment (bproffitt) [reply source/documentation/install-guide/chap-Introduction_to_Hosts.html.md]: @sandrobonazzola Ping

**Pull request merged**